examples/instruct_pix2pix/README.md

```diff
@@ -0,0 +1 @@
+# Training InstructPix2Pix
```

Comment:
Let's maybe add a nice description here and add a link such as this one: https://wandb.ai/sayakpaul/uncategorized/reports/Weave-test_table-23-02-24-11-03-40---VmlldzozNjMyODk1 

We can then let the community try to improve the script on larger training runs